src/frontend/src/pages/HomePage/Home.tsx

```diff
@@ -18,7 +18,7 @@ const Home = () => {
   const history = useHistory();
   const { isLoading, isError, error, data: userSettingsData } = useSingleUserSettings(user.userId);
 
-  if (isLoading) return <LoadingIndicator />;
+  if (isLoading || !userSettingsData) return <LoadingIndicator />;
```

Comment:
lmao they literally cant use it no matter what

Reply:
lol yeah this block will never occur anymore but whatever